auslib/test/admin/views/test_rules.py

```diff
@@ -665,51 +665,66 @@ class TestRuleScheduledChanges(ViewTest):
 
     def setUp(self):
         super(TestRuleScheduledChanges, self).setUp()
+        # TODO: make sure history tables return conditions columns too
```

Comment:
I think this is covered in testGetScheduledChangeHistoryRevisions actually? This was a reminder to myself mid-refactor that I forgot to remove, heh.